README.adoc

```diff
@@ -219,6 +219,28 @@ build:
         - merged-gl-code-quality-report.json
 ----
 
+== CLI usage
+
+The plugin may also be used and configured using the Maven CLI. Available configuration properties are:
+
+* `glcqp.spotbugsEnabled`
+* `glcqp.spotbugsInputFile`
+* `glcqp.checkstyleEnabled`
+* `glcqp.checkstyeInputFile`
+* `glcqp.outputFile`
+
+They are used like this:
+
+[source,shell]
```

Comment:
```suggestion
[source,shell,subs="+attributes"]
```

This should fix the issue that the placeholder is not replaced:

![image](https://github.com/chkal/gitlab-code-quality-plugin/assets/159245/35e6fd02-0503-42f6-bf7c-a5a9d88f6416)

Reply:
Never used AsciiDoc before, learning never stops.